### ISSUE TITLE
why alpha=1 in `hard_rgb_blend` function?

### ISSUE THREAD

**Issue by monniert** [facebookresearch/pytorch3d#700]
## 🐛 Bugs / Unexpected behaviors

Hey, thanks for this great project! I wonder why `hard_rgb_blend` function located in `pytorch3d.blending` returns a constant alpha channel equals to 1? I feel like it would make much more sense to return the binary mask corresponding to the rendered object. Besides:

1. it would be consistent with the `softmax_rgb_blend` case which returns a soft mask corresponding to the object location in the image, 
2. it can be easily computed by replacing line 61 with `alpha = (1 - is_background.type_as(colors).to(device)).unsqueeze(-1)`

Thanks!

**Comment by nikhilaravi** [comment]
@monniert, sure we can add this.

**Comment by nikhilaravi** [comment]
This has now been fixed.

**Comment by monniert** [comment]
awesome thanks a lot!
